daemon/graphdriver/aufs/aufs.go

```diff
@@ -264,6 +265,23 @@ func (a *Driver) createDirsFor(id string) error {
 	return nil
 }
 
+// Helper function to debug EBUSY errors on remove.
+func debugEBusy(mountPath string) {
```

Comment:
I'll make it Info level, since we want to know to get info from the field from customers who run into this and they might not be running daemon in debug mode.